radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetworkController.java

```diff
@@ -58,14 +61,19 @@ public RadixNetworkController build() {
 
 	private final Subject<RadixNodeAction> nodeActions = PublishSubject.<RadixNodeAction>create().toSerialized();
 
+	private final Observable<RadixNodeAction> reducedNodeActions;
+
+	// TODO: Move this into a proper reducer framework
+	private final CopyOnWriteArrayList<Consumer<RadixNodeAction>> reducers = new CopyOnWriteArrayList<>();
+
 	private RadixNetworkController(RadixNetwork network, List<RadixNetworkEpic> epics) {
 		Objects.requireNonNull(network);
 		Objects.requireNonNull(epics);
 
 		this.networkState = BehaviorSubject.createDefault(new RadixNetworkState(Collections.emptyMap()));
 
 		// Run reducers first
-		ConnectableObservable<RadixNodeAction> reducedNodeActions = nodeActions.doOnNext(action -> {
+		final ConnectableObservable<RadixNodeAction> reducedNodeActions = nodeActions.doOnNext(action -> {
```

Comment:
Can the name of this local variable be changed?  It matches the field name at line 64 here, which is a bit awkward.

I'm not really sure why it's not directly assigned to the field in the first place, given that this is the constructor?

Reply:
Because the I only want to expose the `connect` interface in the constructor (aka difference between `ConnectableObservable` in the constructor and `Observable` in the class. Any good ideas in the best way of implementing this in cleanest way?

Reply:
Ah, OK.  If that's the case, then maybe just a rename of the local variable will be fine. Maybe `connectableReducedNodeActions`?